docs/getting_started.rst

```diff
@@ -14,13 +14,23 @@ Install with Conda
 ------------------
 .. _install-with-conda:
 
-`conda <http://www.continuum.io/blog/conda>`_ is a python package manager built for scientific python. Unlike ``easy_install`` or ``pip``, it handles binaries and binary dependencies, which are critical for most scientific workflows. If you're a ``conda`` user, you can install OpenPathSampling by adding the omnia channel. If you're not a conda user, you should look into it. ::
+We recommend using `conda` to install OpenPathSampling.  `conda
+<http://www.continuum.io/blog/conda>`_ is a python package manager built for
+scientific python. Unlike ``easy_install`` or ``pip``, it handles binaries
+and binary dependencies, which are critical for most scientific workflows.
+If you're a ``conda`` user, you can install OpenPathSampling by adding the
+omnia channel. ::
 
-To install OpenPathSampling with conda, use the following commands ::
+To install the most recent release of OpenPathSampling with conda, use the
+following commands ::
 
   $ conda config --add channels http://conda.binstar.org/omnia
```

Comment:
My point is that our Travis tests don't check whether you can actually `conda install` OPS, because they don't use the `meta.yaml` that conda uses when it installs stuff. At least, that's how I understand the fact that @jchodera got a "missing matplotlib" complaint.

The problem is that `conda-requirements` is not tied to `conda/meta.yaml`. Is there a way to fix that? If I'm understanding correctly, `conda-requirements` are not actually requirements, and not actually used by conda directly. They're "things we install using conda on Travis before we run OPS." Does conda itself ever make use of the `conda-requirements` file, or only the `meta.yaml`? Don't we want to actually install using the stuff in `meta.yaml` and make sure that we can run based on that?

My thought with "recommended packages" is that these would be other packages that aren't part of the conda `meta.yaml` but are part of the test suite. We might want to separate the testing into stages (or even separate tests) to make sure that all works correctly.

Short version:
* Currently, our tests don't check whether the conda installation will actually work. From what John had, it appears that if a user just tried to install OPS, with nothing else, the result **would not work**. We need to fix that. What's the best way to fix it?
* We will have some packages that are truly *required* for OPS to run, and some others that we want to include in testing to check that other features work. How do we manage that?